evaluator/utils/packmol.py

```diff
@@ -541,7 +541,7 @@ def pack_box(
     number_of_copies,
     structure_to_solvate=None,
     center_solute=True,
-    tolerance=2.0 * unit.angstrom,
+    tolerance=2.4 * unit.angstrom,
```

Comment:
I think in general 2.0 is an ok default for most applications, so it would be good to change this back.

```suggestion
    tolerance=2.0 * unit.angstrom,
```

Instead I have exposed this as [an input to the corresponding workflow protocol](https://github.com/openforcefield/openff-evaluator/pull/236), so this can be varied using that.

Reply:
Thank you, I have reverted the tolerance back to 2.0 angstrom.